src/main/java/io/fusionauth/samlv2/domain/KeyLocation.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2023, Inversoft Inc., All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+package io.fusionauth.samlv2.domain;
+
+/**
+ * Location for the encrypted symmetric key in relation to the {@code <EncryptedData>} element.
+ *
+ * @author Spencer Witt
+ */
+public enum KeyLocation {
```

Comment:
Open to naming suggestions for these enum values. The two possible places to put the encrypted symmetric key in the SAML response are:
* As a child of the `EncryptedData` containing the encrypted assertion value
* As a sibling of the `EncryptedData` containing the encrypted assertion value.

A partial XML document for the first case is:
```
<EncryptedAssertion>
  <EncryptedData>
    <KeyInfo>
      <EncryptedKey>
        ...
     </EncryptedKey>
    </KeyInfo>
  </EncryptedData>
</EncryptedAssertion>
```

The second case is:
```
<EncryptedAssertion>
  <EncryptedData>...</EncryptedData>
  <EncryptedKey>...</EncryptedKey>
</EncryptedAssertion>
```